docs/DevGuide/ImportingData.md

```diff
@@ -0,0 +1,60 @@
+\cond NEVER
```

Comment:
This should be in `docs/Tutorials`

Reply:
Isn't `Tutorials` for users of our (compiled) code and `DevGuide` for developers? I think this should be where "Creating executables" is because it is about adding importing functionality to an executable. Does that make sense?

Reply:
No, this is code that can easily adjust the compile-time input file (metavariables), it does not go over how to write a new importer. "Creating executables" is writing a new executable from scratch, not enabling a feature we provide.

Reply:
Hm, I don't expect that users of executables will have to worry about this code much. I'll add support for importing numeric initial data to the DgElementArray and the GH executable once this PR is merged. Then `NumericInitialData` will be treated just like we currently handle analytic solutions and data: CMake passes it as the `InitialData` template parameter to the Metavariables. So the user won't need to change anything by hand.
Once executables support numeric initial data, how about we add another user tutorial how to get data into the spectre volume data format so it can be used as initial data?

Reply:
Just so understand, you are envisioning some more global way of enable switching between numerical and analytic initial data? What do you think that will look like?

Reply:
@kidder any thoughts on this?

Reply:
Here's the code for the GH executable that I intend to submit once this PR is merged: https://github.com/nilsleiffischer/spectre/commit/810c9064ae9fc064e65ee8aaa49a9df38e950b73
The link is to the last commit on my [import_num_initial_data_in_gh](https://github.com/nilsleiffischer/spectre/commits/import_num_initial_data_in_gh) branch. It's probably easier to discuss this in a call :)

Reply:
Brief overview: The evolution Metavariables already take `InitialData` as a template parameter. It's currently either an analytic solution or analytic data class, but now it can also be `NumericInitialData`. A bunch of compile-time switches (like the ones we already have for analytic solution/data) determine if the importer parallel component should be part of the component_list, for example. The DgElementArray component is capable of importing numeric initial data and does so if requested.